rdmd.d

```diff
@@ -1,3 +1,4 @@
+#!/usr/bin/env rdmd
```

Comment:
Um... yeah...

Reply:
?
This is just a convenience option and works as long as you have a rdmd binary in your path or did you run into an issue with this?

Reply:
Seems peculiar to give rdmd a shebang to run it via rdmd.

Perhaps `dmd -run` would be more apt?

Reply:
AFAICT you can't specify options on the shebang line.

Reply:
> AFAICT you can't specify options on the shebang line.

If that were so, wouldn't `env` fail to work?

Or does that have something to do with `-run` needing to be the last argument?

Reply:
> If that were so, wouldn't env fail to work?

Oh I meant that the Shebang line only accepts one optional argument.
See also:

https://en.wikipedia.org/wiki/Shebang_(Unix)
http://lists.gnu.org/archive/html/bug-sh-utils/2002-04/msg00020.html

Reply:
Ah, TIL. Thanks.